src/test/java/org/fogbowcloud/manager/requests/api/local/http/AttachmentOrdersControllerTest.java

```diff
@@ -0,0 +1,395 @@
+    // Check the response of request and the call of facade for get the attachments. 
+    @Test
+    public void getAllAttachmentsWhenHasNoData() throws Exception {
+    	//set up
+        List<AttachmentInstance> attachementInstanceList = new ArrayList<>();
+        Mockito.doReturn(attachementInstanceList).when(this.facade).getAllAttachments(Mockito.anyString());
+
+        RequestBuilder requestBuilder =
+                createRequestBuilder(HttpMethod.GET, ATTACHMENT_ENDPOINT, getHttpHeaders(), "");
+
+        // exercise: Make the request
+        MvcResult result = this.mockMvc.perform(requestBuilder).andReturn();
+
+        // verify
+        int expectedStatus = HttpStatus.OK.value();
+        Assert.assertEquals(expectedStatus, result.getResponse().getStatus());
+        
+        String expectedResult = "[]";
+
+        Assert.assertEquals(expectedResult, result.getResponse().getContentAsString());
+        
+        Mockito.verify(this.facade, Mockito.times(1))
+			.getAllAttachments(Mockito.anyString());
+    }
+
+    // test case: Request the list of all attachments when the facade returns an non-empty list. 
+    // Check the response of request and the call of facade for get the attachments. 
+    @Test
+    public void getAllAttachmentsWhenHasData() throws Exception {
+    	// set up
+        AttachmentInstance AttachmentInstance1 = new AttachmentInstance("fake-Id-1");
+        AttachmentInstance AttachmentInstance2 = new AttachmentInstance("fake-Id-2");
+        AttachmentInstance AttachmentInstance3 = new AttachmentInstance("fake-Id-3");
+
+        List<AttachmentInstance> AttachmentInstanceList =
+                Arrays.asList(
+                        new AttachmentInstance[] {
+                            AttachmentInstance1, AttachmentInstance2, AttachmentInstance3
+                        });
+        Mockito.doReturn(AttachmentInstanceList).when(this.facade)
+        	.getAllAttachments(Mockito.anyString());
+
+        RequestBuilder requestBuilder =
+                createRequestBuilder(HttpMethod.GET,
+                		ATTACHMENT_ENDPOINT,
+                		getHttpHeaders(), "");
+        
+        // exercise: Make the request.
+        MvcResult result = this.mockMvc.perform(requestBuilder).andReturn();
+
+        // verify
+        int expectedStatus = HttpStatus.OK.value();
+        Assert.assertEquals(expectedStatus, result.getResponse().getStatus());
+
+        TypeToken<List<AttachmentInstance>> token = new TypeToken<List<AttachmentInstance>>() {};
+        List<AttachmentInstance> resultList =
+                new Gson().fromJson(result.getResponse().getContentAsString(), token.getType());
+        Assert.assertTrue(resultList.size() == 3);
+        
+        Mockito.verify(this.facade, Mockito.times(1))
+			.getAllAttachments(Mockito.anyString());
+    }
+    
+    // test case: Request the list of all attachments status when the facade returns an non-empty list. 
```

Comment:
I'm not sure but it seems that most of these test cases are related to asserting that the returned json is compatible with the values return by the facade (which is a collaborator of the controller). If so, tell this in the test cases description, it would be clearer.